tools/crm_mon_output.c

```diff
@@ -0,0 +1,436 @@
+/*
+ * Copyright 2019 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+ */
+
+#include <glib.h>
+#include <stdarg.h>
+
+#include <crm/stonith-ng.h>
+#include <crm/common/iso8601_internal.h>
+#include <crm/common/output.h>
+#include <crm/common/util.h>
+#include <crm/common/xml.h>
+
+#include "crm_mon.h"
+
+typedef void (*print_fn_ty)(pcmk__output_t *out, const char *format, ...) G_GNUC_PRINTF(2, 3);
+
+static print_fn_ty
+get_print_fn(pcmk__output_t *out) {
+    if (safe_str_eq(out->fmt_name, "console")) {
+        return curses_indented_printf;
+    } else {
+        return pcmk__indented_printf;
+    }
```

Comment:
Fixing that up was way easier than I thought it'd be.  I must have thought it would be a lot more useful in the beginning, but it didn't end up that way.  The duplication is not too bad.